source/blog/2016-08-15-last-week-in-manageiq.html.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Last Week in ManageIQ: About modal, gettext improvements, refactoring and more!"
+date: 2016-08-15 06:20 UTC
+author: skateman
+comments: true
+published: true
+---
+
+Hi! [David](https://twitter.com/halaszdavid) here, from the UI team!
+
+Let's take a quick review about the changes to ManageIQ since last week.
+
+## Featured
+
+The last week [114 merged pull requests][PRs merged last week] have been
```

Comment:
Possibly "In the last week" or "The last week has seen [...] pull requests produced"?